packages/vue/src/templates/edu/PageEduLesson/PageEduLesson.stories.js

```diff
@@ -169,7 +169,49 @@ export const BaseStory = {
 
       procedures: [
         {
-          blocks: BlockStreamfieldMinimalData.body
+          sectionHeading: 'Section Heading 1',
+          stepsNumbering: true,
+          steps: [
+            {
+              blocks: BlockStreamfieldMinimalData.body
+            },
+            {
+              blocks: BlockStreamfieldMinimalData.body
+            },
+            {
+              blocks: BlockStreamfieldMinimalData.body
+            }
+          ]
+        },
+        {
+          // sectionHeading: 'Section Heading 2',
+          stepsNumbering: true,
+          steps: [
+            {
+              blocks: BlockStreamfieldMinimalData.body
+            },
+            {
+              blocks: BlockStreamfieldMinimalData.body
+            },
+            {
+              blocks: BlockStreamfieldMinimalData.body
+            }
+          ]
+        },
+        {
+          sectionHeading: 'Section Heading 3',
+          stepsNumbering: true,
```

Comment:
Let's test this one with no numbering.

```suggestion
          stepsNumbering: false,
```